Makefile

```
@@ -119,7 +119,7 @@ RSRCS = \
 #	- 	if your library does not follow the standard library naming scheme,
 #		you need to specify the path to the library and it's name.
 #		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
-LIBS = be tracker translation sqlite3 columnlistview $(STDCPPLIBS)
+LIBS = be tracker localestub sqlite3 columnlistview $(STDCPPLIBS)
```

Still needs the "translation" kit for GetBitmap()

```suggestion
LIBS = be tracker localestub translation sqlite3 columnlistview $(STDCPPLIBS)
```